### PR TITLE
Update readme (again) and issue template

### PR DESCRIPTION
**Pull request by Margen67**
Added Discord link to issue template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,35 +16,36 @@ We now boot more titles than ever before, with more and more reaching gameplay.
 
 ## System Requirements
 ### Minimum
-  * OS: Windows 7 64-bit or newer. 32-bit installations are not supported.
-  * Video card: anything that supports Direct3D 9 (HLE) and OpenGL 3.3 (LLE).
+  * OS: Windows 7+ x64. 32-bit installations are not supported.
+  * GPU: Anything that supports Direct3D 9 (HLE) and OpenGL 3.3 (LLE).
+  * CPU: x64 with SSE2.
```
Comment (LukeUsher) [inline README.md]: It is redundant to specify this as '32-bit installations are not supported' implies that X64 support is required, and there are no x64 CPUs without SSE2, at all.

File: README.md [hunk 2]
```diff
@@ -15,35 +16,36 @@ We now boot more titles than ever before, with more and more reaching gameplay.
...
 ### Prerequisites
-  * Visual C++ 2015 and 2017 redistributables may be required. Download them [here](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads).
+  * Visual C++ 2015 and 2017 redistributables may be needed. Download them [here](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads).
```
Comment (LukeUsher) [inline README.md]: Why this wording change? (required->needed)

File: README.md (hunk 1 above)
Comment (Margen67) [reply README.md]: Should I replace it with a clock speed? (like 1.5GHz)

Comment (LukeUsher) [reply README.md]: There is currently no known minimum CPU requirements, and it's likely to change very regularly as we implement more functionality and optimize, probably best to leave CPU out completely

File: README.md (hunk 2 above)
Comment (Margen67) [reply README.md]: They both mean the same thing so I'll just undo it.

**Comment by Margen67** [comment]
@LukeUsher Is it good now?

**Pull request merged**